ios/FluentUI.Demo/FluentUI.Demo/Demos/ListItemDemoController_SwiftUI.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import SwiftUI
+
+typealias ListItemSampleData = TableViewCellSampleData
+
+class ListItemDemoControllerSwiftUI: UIHostingController<ListItemDemoView> {
```

Comment:
Have you confirmed that `UIHostingController` is wrapping subviews correctly (just want to confirm that a bug like the AvatarGroup one is fixed)